numalogic/udfs/preprocess.py

```diff
@@ -151,12 +189,34 @@ def exec(self, keys: list[str], datum: Datum) -> Messages:
             )
             # TODO check again what error is causing this and if retraining is required
             payload = replace(payload, status=Status.RUNTIME_ERROR, header=Header.TRAIN_REQUEST)
+            _increment_counter(
+                counter=MODEL_STATUS_COUNTER,
+                labels=(
+                    payload.status.value,
+                    self._vtx,
+                    ":".join(payload.composite_keys),
+                    payload.config_id,
```

Comment:
```suggestion
                    payload.status.value,
                    *_label_values,
```
And define the `_label_values` as a local variable in the exec(). Seems we are adding this same subset many times.

Reply:
Yes a very good idea! Let me make that